packages/ckeditor5-emoji/ckeditor5-metadata.json

```
@@ -0,0 +1,10 @@
+{
+	"plugins": [
+		{
+			"name": "Emoji",
+			"className": "Emoji",
+			"description": "Allows for inserting emojis via a dropdown or by typing `:` followed by emoji name.",
+			"path": "src/emoji.js"
```

Docs path is already mentioned in several places. Let's add it to the metadata as well so we don't forget about it.

```suggestion
			"path": "src/emoji.js",
			"docs": "features/emoji.html"
```